### DISCUSSION TITLE
How to redirect to home page ('/') when 404 unmatched URLs on app router

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#55421] [App Router]
I have a doubt that, how can we redirect to home page ('/') when 404 unmatched URLs on app router?

**Comment by devjiwonchoi** [comment]
I'd suggest using the global not-found as:

```ts
// app/not-found.tsx
import { redirect } from 'next/navigation'

export default function GlobalNotFound() {
  redirect('/')
}
```

**Reply by ghost** [reply]
@devjiwonchoi Thanks for your response it worked, but is there a way we could do with redirection on next config? or with middleware?
